desc/compute/_qs.py

```diff
@@ -301,6 +422,47 @@ def _B_modes(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Boozer transform modes norm",
+    label="\\nu_{mn} = (\\zeta_{B} - \\zeta)_{mn}",
+    units="rad",
+    units_long="radians",
```

Comment:
double check these, i think it should be unitless, and can probably leave the label blank

Reply:
forgot to change these, thakns for the catch